### PR TITLE
"navBar & Footer & review section"

### PR DESCRIPTION
**Pull request by SAMIYAghb**

### CODE REVIEW

**Comment by Ahmed-Abou-Emran** [comment]
Great Job !
I am just trying to open the application, but it seems that the code is not updated, since it contains a lot of errors and missing files.
Also the name of the branch `userHeader` doesn't give an overview of what is happening in the branch, but it currently contains the navbar - footer, and reviews section.

image

File: src/features/User/Ui/Review/Review.tsx [hunk 1]
```diff
@@ -1,9 +1,66 @@
...
+      maxWidth="md">
+        <Stack
+          spacing={{ xs: 1, sm: 2 }}
+          direction={{ xs: "column", md: "row" }}
+          useFlexGap
+          // flexWrap="wrap"
+          justifyContent="center"
+          alignItems="center"
+        >
+          <Box className={styles.reviewImageParent}>
```
Comment (Ahmed-Abou-Emran) [inline src/features/User/Ui/Review/Review.tsx:37]: The Image banner should the same dimensions as the image
You can create a div that will wrap the image and make the image take its dimensions and then move the image using `transform:translate()`.

Here is an example of what I mean:
https://codepen.io/y-ahmed-ali-y/pen/gOEWLBb

File: src/features/User/Ui/Footer/Footer.tsx [hunk 2]
```diff
@@ -1,9 +1,113 @@
...
+      ],
+    },
+  ];
+
   return (
-    <div>Footer</div>
-  )
-}
+    <ScopedCssBaseline  sx={{ backgroundColor: "#f5f5f5" }}>
+    <Container component="footer">
```
Comment (Ahmed-Abou-Emran) [inline src/features/User/Ui/Footer/Footer.tsx:38]: Footer elements are not aligned at top
Also lists in the footer are not aligned with their corresponding headers

image

File: src/features/User/Ui/Footer/Footer.tsx [hunk 3]
```diff
@@ -1,9 +1,113 @@
...
+              {section.title}
+            </Typography>
+            {section.links && (
+              <List>
+                {section.links.map((link, linkIndex) => (
+                  <ListItem
+                    key={linkIndex}
+                    sx={{ fontWeight: "bold", color: "text.secondary" }}
+                  >
+                    {link.text}
```
Comment (Ahmed-Abou-Emran) [inline src/features/User/Ui/Footer/Footer.tsx:88]: You need to make them as an actual link, and if their path is not implemented yet, we can make them refer to the current page for now.

File: src/features/User/Ui/NavBar/NavBar.tsx [hunk 4]
```diff
@@ -1,45 +1,140 @@
...
+                  id="fade-menu"
+                  MenuListProps={{
+                    "aria-labelledby": "fade-button",
+                  }}
+                  anchorEl={anchorEl}
+                  open={open}
+                  onClose={handleClose}
+                  TransitionComponent={Fade}
+                >
+                  <MenuItem onClick={handleClose}>Profile</MenuItem>
```
Comment (Ahmed-Abou-Emran) [inline src/features/User/Ui/NavBar/NavBar.tsx:126]: There are additional navigations and a login button added to the Figma design, you might need to consider it in the design.

image

**Review by Ahmed-Abou-Emran — CHANGES_REQUESTED** [review]
Great job on the work so far! I appreciate your effort . I've left few comments highlighting areas that may need a little adjustment. Take your time to review them , and once you've made the necessary changes, feel free to request a review again.

File: src/features/User/Ui/Review/Review.tsx (hunk 1 above)
Comment (SAMIYAghb) [reply src/features/User/Ui/Review/Review.tsx:37]: done

File: src/features/User/Ui/Footer/Footer.tsx (hunk 2 above)
Comment (SAMIYAghb) [reply src/features/User/Ui/Footer/Footer.tsx:38]: done

**Review by Ahmed-Abou-Emran — CHANGES_REQUESTED** [review]
Required Changes:

- [x] Header colors : logo, text, background doesn't match the design.
- [x] Footer lists should be aligned to the left, you just need to remove the padding of the `li` items.
- [x] The Reviews section is taking a background color that is different than the one in the design.

**Pull request merged**